### PR TITLE
Fix corrupted db snapshot

### LINKED ISSUE #1926: Snapshot doesn't work
Hello,
I have downloaded snapshot from https://juno-snapshots.nethermind.dev/mainnet/juno_mainnet_v0.11.9_654464.tar and trying to use the snapshot within a k8s cluster manually, after I unarchive the snapshot to the pvc directly and restarted the POD, it shows me error as bellow:
```
kubectl logs starknet-mainnet-0 -f
Defaulted container "starknet-mainnet" out of: starknet-mainnet, init (init)

       _
      | |
      | |_   _ _ __   ___
  _   | | | | | '_ \ / _ \
 | |__| | |_| | | | | (_) |
  \____/ \__,_|_| |_|\___/ v0.11.9

Juno is a Go implementation of a Starknet full-node client created by Nethermind.

Error: open DB: L0: 9168006: file 9168006 (type 2) unknown to the objstorage provider: file does not exist
L0: 9168005: file 9168005 (type 2) unknown to the objstorage provider: file does not exist
L2: 9168008: file 9168008 (type 2) unknown to the objstorage provider: file does not exist
L2: 9168009: file 9168009 (type 2) unknown to the objstorage provider: file does not exist
L2: 9168010: file 9168010 (type 2) unknown to the objstorage provider: file does not exist
L2: 9168011: file 9168011 (type 2) unknown to the objstorage provider: file does not exist
L2: 9168012: file 9168012 (type 2) unknown to the objstorage provider: file does not exist
L2: 9168013: file 9168013 (type 2) unknown to the objstorage provider: file does not exist
L2: 9168014: file 9168014 (type 2) unknown to the objstorage provider: file does not exist
L2: 9167979: file 9167979 (type 2) unknown to the objstorage provider: file does not exist
L2: 9167983: file 9167983 (type 2) unknown to the objstorage provider: file does not exist
L2: 9167985: file 9167985 (type 2) unknown to the objstorage provider: file does not exist
L2: 9167987: file 9167987 (type 2) unknown to the objstorage provider: file does not exist
L2: 9167988: file 9167988 (type 2) unknown to the objstorage provider: file does not exist
L2: 9168015: file 9168015 (type 2) unknown to the objstorage provider: file does not exist
L2: 9168016: file 9168016 (type 2) unknown to the objstorage provider: file does not exist
L2: 9168022: file 9168022 (type 2) unknown to the objstorage provider: file does not exist
Usage:
  juno [flags]
  juno [command]

Available Commands:
  completion    Generate the autocompletion script for the specified shell
  genp2pkeypair Generate private key pair for p2p.
  help          Help about any command

Flags:
      --cn-core-contract-address string   Custom network core contract address.
      --cn-feeder-url string              Custom network feeder URL.
      --cn-gateway-url string             Custom network gateway URL.
      --cn-l1-chain-id string             Custom network L1 chain id.
      --cn-l2-chain-id string             Custom network L2 chain id.
      --cn-name string                    Custom network name.
      --cn-unverifiable-range 0,100       Custom network range of blocks to skip hash verifications (e.g. 0,100).
      --colour --colour=false             Use --colour=false command to disable colourized outputs (ANSI Escape Codes). (default true)
      --config string                     The YAML configuration file.
# Please edit the object below. Lines beginning with a '#' will be ignored,
      --db-cache-size uint                Determines the amount of memory (in megabytes) allocated for caching data in the database. (default 8)
      --db-max-handles int                A soft limit on the number of open files that can be used by the DB (default 1024)
      --db-path string                    Location of the database files. (default "/juno")
      --eth-node string                   WebSocket endpoint of the Ethereum node. To verify the correctness of the L2 chain, Juno must connect to an Ethereum node and parse events in the Starknet contract.
# Please edit the object below. Lines beginning with a '#' will be ignored,
      --grpc                              Enable the HTTP gRPC server on the default port.
      --grpc-host string                  The interface on which the gRPC server will listen for requests. (default "localhost")
      --grpc-port uint16                  The port on which the gRPC server will listen for requests. (default 6064)
      --gw-api-key string                 API key for gateway endpoints to avoid throttling
      --gw-timeout duration               Timeout for requests made to the gateway (default 5s)
  -h, --help                              help for juno
      --http                              Enables the HTTP RPC server on the default port and interface.
      --http-host string                  The interface on which the HTTP RPC server will listen for requests. (default "localhost")
      --http-port uint16                  The port on which the HTTP server will listen for requests. (default 6060)
      --log-level LogLevel                Options: trace, debug, info, warn, error. (default info)
      --max-vm-queue uint                 Maximum number for requests to queue after reaching max-vms before starting to reject incoming requests (default 24)
      --max-vms uint                      Maximum number for VM instances to be used for RPC calls concurrently (default 12)
      --metrics                           Enables the Prometheus metrics endpoint on the default port.
      --metrics-host string               The interface on which the Prometheus endpoint will listen for requests. (default "localhost")
      --metrics-port uint16               The port on which the Prometheus endpoint will listen for requests. (default 9090)
      --network Network                   Options: mainnet, sepolia, sepolia-integration. (default mainnet)
      --p2p                               EXPERIMENTAL: Enables p2p server.
      --p2p-addr string                   EXPERIMENTAL: Specify p2p source address as multiaddr.
      --p2p-feeder-node                   EXPERIMENTAL: Run juno as a feeder node which will only sync from feeder gateway and gossip the new blocks to the network.
      --p2p-peers string                  EXPERIMENTAL: Specify list of p2p peers split by a comma. These peers can be either Feeder or regular nodes.
      --p2p-private-key string            EXPERIMENTAL: Hexadecimal representation of a private key on the Ed25519 elliptic curve.
      --pending-poll-interval duration    Sets how frequently pending block will be updated (0s will disable fetching of pending block). (default 5s)
      --pprof                             Enables the pprof endpoint on the default port.
      --pprof-host string                 The interface on which the pprof HTTP server will listen for requests. (default "localhost")
      --pprof-port uint16                 The port on which the pprof HTTP server will listen for requests. (default 6062)
      --remote-db string                  gRPC URL of a remote Juno node
      --rpc-call-max-steps uint           Maximum number of steps to be executed in starknet_call requests. The upper limit is 4 million steps, and any higher value will still be capped at 4 million. (default 4000000)
      --rpc-cors-enable                   Enable CORS on RPC endpoints
      --rpc-max-block-scan uint           Maximum number of blocks scanned in single starknet_getEvents call (default 18446744073709551615)
  -v, --version                           version for juno
      --ws                                Enables the WebSocket RPC server on the default port.
      --ws-host string                    The interface on which the WebSocket RPC server will listen for requests. (default "localhost")
      --ws-port uint16                    The port on which the WebSocket server will listen for requests. (default 6061)

Use "juno [command] --help" for more information about a command.
```

Here are the pvc file list 
image
image

And this is POD description:
```
kubectl describe pods starknet-mainnet-0
Name:             starknet-mainnet-0
Namespace:        nodes
Priority:         0
Service Account:  default
Node:             sea-p-3/10.16.0.217
Start Time:       Fri, 05 Jul 2024 09:55:03 +0800
Labels:           app=starknet-mainnet
                  apps.kubernetes.io/pod-index=0
                  controller-revision-hash=starknet-mainnet-86d47799d8
                  statefulset.kubernetes.io/pod-name=starknet-mainnet-0
Annotations:      cni.projectcalico.org/containerID: e3a72306c1de32740be12e3a1a64b770a1d89eb712cef9f648284ae3ad6b1626
                  cni.projectcalico.org/podIP: 10.234.248.28/32
                  cni.projectcalico.org/podIPs: 10.234.248.28/32
                  prometheus.io/path: /metrics
                  prometheus.io/port: 9090
                  prometheus.io/scrapenode: true
Status:           Running
IP:               10.234.248.28
IPs:
  IP:           10.234.248.28
Controlled By:  StatefulSet/starknet-mainnet
Init Containers:
  init:
    Container ID:  containerd://96a784775d559e5932c40dab3a188c2130ef474cce5192115fe4cd88b1248438
    Image:         ubuntu:latest
    Image ID:      docker.io/library/ubuntu@sha256:2e863c44b718727c860746568e1d54afd13b2fa71b160f5cd9058fc436217b30
    Port:          <none>
    Host Port:     <none>
    Command:
      /bin/sh
      -c
    Args:
      echo "Syncing without Snapshot"
    State:          Terminated
      Reason:       Completed
      Exit Code:    0
      Started:      Fri, 05 Jul 2024 09:55:05 +0800
      Finished:     Fri, 05 Jul 2024 09:55:05 +0800
    Ready:          True
    Restart Count:  0
    Environment:    <none>
    Mounts:
      /var/lib/juno from starknet-mainnet-pv (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-k5znz (ro)
Containers:
  starknet-mainnet:
    Container ID:  containerd://7d2e20901706214fcea487c9d24b1b73dbcbec72ead0b276defceb4285e6bc69
    Image:         nethermind/juno:v0.11.9
    Image ID:      docker.io/nethermind/juno@sha256:9d172566dc090e0e8a629f6526a82e64bc201dba85f1333ad1f7d9d6bafa7762
    Ports:         6060/TCP, 6061/TCP
    Host Ports:    0/TCP, 0/TCP
    Args:
      --db-path=/var/lib/juno
      --eth-node=ws://ethereum:8545
      --http=true
      --http-host=0.0.0.0
      --http-port=6060
      --metrics=true
      --metrics-host=0.0.0.0
      --metrics-port=9090
      --network=mainnet
      --pending-poll-interval=2s
      --ws=true
      --ws-host=0.0.0.0
      --ws-port=6061
    State:          Running
      Started:      Fri, 05 Jul 2024 09:55:06 +0800
    Ready:          True
    Restart Count:  0
    Limits:
      cpu:     4
      memory:  14Gi
    Requests:
      cpu:        2
      memory:     8Gi
    Environment:  <none>
    Mounts:
      /var/lib/juno from starknet-mainnet-pv (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-k5znz (ro)
Conditions:
  Type                        Status
  PodReadyToStartContainers   True
  Initialized                 True
  Ready                       True
  ContainersReady             True
  PodScheduled                True
Volumes:
  starknet-mainnet-pv:
    Type:       PersistentVolumeClaim (a reference to a PersistentVolumeClaim in the same namespace)
    ClaimName:  starknet-mainnet-pv-starknet-mainnet-0
    ReadOnly:   false
  kube-api-access-k5znz:
    Type:                    Projected (a volume that contains injected data from multiple sources)
    TokenExpirationSeconds:  3607
    ConfigMapName:           kube-root-ca.crt
    ConfigMapOptional:       <nil>
    DownwardAPI:             true
QoS Class:                   Burstable
Node-Selectors:              kubernetes.io/hostname=sea-p-3
Tolerations:                 node.kubernetes.io/not-ready:NoExecute op=Exists for 300s
                             node.kubernetes.io/unreachable:NoExecute op=Exists for 300s
Events:
  Type    Reason     Age   From               Message
  ----    ------     ----  ----               -------
  Normal  Scheduled  25m   default-scheduler  Successfully assigned nodes/starknet-mainnet-0 to sea-p-3
  Normal  Pulling    25m   kubelet            Pulling image "ubuntu:latest"
  Normal  Pulled     25m   kubelet            Successfully pulled image "ubuntu:latest" in 1.356s (1.356s including waiting)
  Normal  Created    25m   kubelet            Created container init
  Normal  Started    25m   kubelet            Started container init
  Normal  Pulling    25m   kubelet            Pulling image "nethermind/juno:v0.11.9"
  Normal  Pulled     25m   kubelet            Successfully pulled image "nethermind/juno:v0.11.9" in 596ms (596ms including waiting)
  Normal  Created    25m   kubelet            Created container starknet-mainnet
  Normal  Started    25m   kubelet            Started container starknet-mainnet
```

### PR DESCRIPTION
**Pull request by wojciechos**
closes #1926

### CODE REVIEW

File: docs/docs/snapshots.md [hunk 1]
```diff
@@ -10,13 +10,13 @@ You can download a snapshot of the Juno database to reduce the network syncing t
 
 | Version      | Size       | Block      | Download Link                                                                                         |
 | ------------ | ---------- | ---------- | ----------------------------------------------------------------------------------------------------- |
-| **>=v0.9.2** | **172 GB** | **654464** | [**juno_mainnet.tar**](https://juno-snapshots.nethermind.dev/mainnet/juno_mainnet_v0.11.9_654464.tar) |
+| **>=v0.9.2** | **172 GB** | **654464** | [**juno_mainnet.tar**](https://juno-snapshots.nethermind.dev/mainnet/juno_mainnet_v0.11.9_654881.tar) |
```
Comment (kirugan) [inline docs/docs/snapshots.md:13]: do we need to change 654464 to 654881 here as well?

File: docs/docs/snapshots.md [hunk 2]
```diff
@@ -10,13 +10,13 @@ You can download a snapshot of the Juno database to reduce the network syncing t
...
 
 ## Sepolia
 
 | Version      | Size     | Block     | Download Link                                                                                        |
 | ------------ | -------- | --------- | ---------------------------------------------------------------------------------------------------- |
-| **>=v0.9.2** | **5 GB** | **66477** | [**juno_sepolia.tar**](https://juno-snapshots.nethermind.dev/sepolia/juno_sepolia_v0.11.7_66477.tar) |
+| **>=v0.9.2** | **5 GB** | **654881** | [**juno_sepolia.tar**](https://juno-snapshots.nethermind.dev/sepolia/juno_sepolia_v0.11.7_66477.tar) |
```
Comment (kirugan) [inline docs/docs/snapshots.md:19]: if snapshot link is the same why do we need to change it here?

Comment (wojciechos) [reply docs/docs/snapshots.md]: Yep, my bad - fixed

File: docs/docs/snapshots.md (hunk 1 above)
Comment (wojciechos) [reply docs/docs/snapshots.md]: Yep, my bad - fixed

**Review by kirugan — APPROVED** [review]

**Review by weiihann — APPROVED** [review]

**Pull request merged**